packages-ts/starknet-gauntlet-emergency-protocol/README.md

```diff
@@ -19,12 +19,16 @@ This deploys a new instance of the `StarknetValidator` contract on **L1**
 `<SOURCE>` is the address of source aggregator. Source aggregator should be added to access control of starknet validator contract to be able to call
 `validate` on the starknet validator
 
-`<GAS_ESTIMATE>` is a number that represents l1 gas estimate
+`<GAS_ESTIMATE>` is a number that represents l1 gas estimate. We recommend a value of 17300 on mainnet. On devnet or testnet you may choose 1 to save gas.
```

Comment:
This depends on the receiving contract, so it would be valuable to add the specific contract version here. Once that contract changes, the gas will value will most likely change.

Reply:
True, since we don't have versions I'll just add something that references the date of writing

Reply:
Hmm, we should have versions for each contract inside the contract like [this](https://github.com/smartcontractkit/chainlink-starknet/blob/develop/contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/sequencer_uptime_feed.cairo#LL78C6-L78C22):

```
@view
func type_and_version{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (
    meta: felt
) {
    let (meta) = SequencerUptimeFeed.type_and_version();
    return (meta,);
}
```

As influenced from here [ITypeAndVersion.sol](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/ITypeAndVersion.sol). Can you please double check and add it to contracts missing? cc @archseer

Reply:
Do we have versions for all contracts? Please double-check.